generated/nidigitalpattern/nidigitalpattern.proto

```diff
@@ -83,40 +57,24 @@ service NiDigital {
   rpc GetHistoryRAMSampleCount(GetHistoryRAMSampleCountRequest) returns (GetHistoryRAMSampleCountResponse);
-  rpc GetPatternName(GetPatternNameRequest) returns (GetPatternNameResponse);
-  rpc GetPatternPinIndexes(GetPatternPinIndexesRequest) returns (GetPatternPinIndexesResponse);
   rpc GetPatternPinList(GetPatternPinListRequest) returns (GetPatternPinListResponse);
-  rpc GetPinName(GetPinNameRequest) returns (GetPinNameResponse);
-  rpc GetPinResultsPinInformation(GetPinResultsPinInformationRequest) returns (GetPinResultsPinInformationResponse);
   rpc GetSitePassFail(GetSitePassFailRequest) returns (GetSitePassFailResponse);
   rpc GetSiteResultsSiteNumbers(GetSiteResultsSiteNumbersRequest) returns (GetSiteResultsSiteNumbersResponse);
   rpc GetTimeSetDriveFormat(GetTimeSetDriveFormatRequest) returns (GetTimeSetDriveFormatResponse);
   rpc GetTimeSetEdge(GetTimeSetEdgeRequest) returns (GetTimeSetEdgeResponse);
   rpc GetTimeSetEdgeMultiplier(GetTimeSetEdgeMultiplierRequest) returns (GetTimeSetEdgeMultiplierResponse);
-  rpc GetTimeSetName(GetTimeSetNameRequest) returns (GetTimeSetNameResponse);
   rpc GetTimeSetPeriod(GetTimeSetPeriodRequest) returns (GetTimeSetPeriodResponse);
-  rpc Init(InitRequest) returns (InitResponse);
   rpc InitWithOptions(InitWithOptionsRequest) returns (InitWithOptionsResponse);
-  rpc Initiate(InitiateRequest) returns (InitiateResponse);
   rpc IsDone(IsDoneRequest) returns (IsDoneResponse);
   rpc IsSiteEnabled(IsSiteEnabledRequest) returns (IsSiteEnabledResponse);
   rpc LoadLevels(LoadLevelsRequest) returns (LoadLevelsResponse);
   rpc LoadPattern(LoadPatternRequest) returns (LoadPatternResponse);
   rpc LoadPinMap(LoadPinMapRequest) returns (LoadPinMapResponse);
   rpc LoadSpecifications(LoadSpecificationsRequest) returns (LoadSpecificationsResponse);
   rpc LoadTiming(LoadTimingRequest) returns (LoadTimingResponse);
-  rpc MapPinToChannel(MapPinToChannelRequest) returns (MapPinToChannelResponse);
-  rpc PPMUConfigureApertureTime(PPMUConfigureApertureTimeRequest) returns (PPMUConfigureApertureTimeResponse);
-  rpc PPMUConfigureCurrentLevel(PPMUConfigureCurrentLevelRequest) returns (PPMUConfigureCurrentLevelResponse);
-  rpc PPMUConfigureCurrentLevelRange(PPMUConfigureCurrentLevelRangeRequest) returns (PPMUConfigureCurrentLevelRangeResponse);
-  rpc PPMUConfigureCurrentLimit(PPMUConfigureCurrentLimitRequest) returns (PPMUConfigureCurrentLimitResponse);
-  rpc PPMUConfigureCurrentLimitRange(PPMUConfigureCurrentLimitRangeRequest) returns (PPMUConfigureCurrentLimitRangeResponse);
-  rpc PPMUConfigureOutputFunction(PPMUConfigureOutputFunctionRequest) returns (PPMUConfigureOutputFunctionResponse);
-  rpc PPMUConfigureVoltageLevel(PPMUConfigureVoltageLevelRequest) returns (PPMUConfigureVoltageLevelResponse);
-  rpc PPMUConfigureVoltageLimits(PPMUConfigureVoltageLimitsRequest) returns (PPMUConfigureVoltageLimitsResponse);
+  rpc LockSession(LockSessionRequest) returns (LockSessionResponse);
```

Comment:
LockSession and UnlockSession should be marked 'codegen_method': 'private' in hapigen.